### DISCUSSION TITLE
How do I get a usable :dig?

### DISCUSSION THREAD

**Discussion by gapato** [LazyVim/LazyVim#1226] [Q&A]
From time to time I need to have a look at :diagraphs / :dig, but with LazyVim the resulting output is stuffed in a notification (see below) instead of being displayed on the entire screen. I suspect this has something to do with nvim-notify, but I do not understand how to disable it for :dig.

Does anyone know how this is done?

![Screenshot_2023-07-26_14-16-46](https://github.com/LazyVim/LazyVim/assets/26614/0468a5f9-aad6-4949-83ae-b9f128d43aa5)

**Comment by dpetka2001** [comment]
`:Telescope notify`?

Edit: or even `<leader>sna` for `Noice all` and then maybe you can do a `<C-w>_` to set the height of the window to maximum, which could be more comfortable I assume?

**Comment by gapato** [comment]
Thank you for your suggestions!

`:Telescope notify` is an unknown command, at least in my install. `Noice all` goes in the right direction, but the alignment is completely messed up, making the page difficult to read.

**Reply by dpetka2001** [reply]
How about doing `:dig!` and then `Noice all`? Does that look better for you?

Edit: I also disabled wrap with `<leader>uw` and I believe it looks better. The columns look more discernible, but you have to scroll horizontally to look at all columns, since wrapping is disabled.
